pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -1140,6 +1138,15 @@ def network_init(self):
             '--num-seats',
             str(n['num_seats']),
         ]
+
+        # Needed for backwards compatibility. Works because 'dump' is the
+        # default option.
+        # TODO: After we stop testing neard binaries with version < 2.7.0, use
+        # the --state-source flag unconditionally.
```

Comment:
How could we ensure we get back to this? Maybe create an issue and reference it here?

Reply:
Well, based on my experience, issues get lost and closed too. I can write `TODO(2.8)` here which means that it can be resolved since 2.8 version.